v1.0/manual-deployment.md

```diff
@@ -86,19 +94,29 @@ Locally, you'll need to [create the following certificates and keys](create-secu
 	# Create the certs directory:
 	$ ssh <username>@<node1 address> "mkdir certs"
 
-	# Upload the CA certificate, client (root) certificate and key, and node certificate and key:
+	# Upload the CA certificate, client certificates and keys, and node certificate and key:
```

Comment:
We don't need to copy the client certificate and key to the node. I can understand `root` being useful for local debugging, but other users really don't need to be there.